modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -48,29 +55,20 @@ Complete the following steps on the installation host:
 
 ifndef::openshift-origin[]
 . Download your `registry.redhat.io` pull secret from the
-link:https://console.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {console-redhat-com} site and save it to a `.json` file.
-endif::[]
-
-. Generate the base64-encoded user name and password or token for your mirror
-registry:
-+
-[source,terminal]
-----
-$ echo -n '<user_name>:<password>' | base64 -w0 <1>
-BGVtbYk3ZHAtqXs=
-----
-<1> For `<user_name>` and `<password>`, specify the user name and password that
-you configured for your registry.
+link:https://console.redhat.com/openshift/install/pull-secret[Pull Secret] page on the {console-redhat-com} site.
```

Comment:
@jpower432 Okay I updated this file if you can please take another look. I swapped steps 2 and 3, fixed a bit of the wording and step 2 command, and then added a different callout for step 2 for oc-mirror about it having to be one two of those options.

Reply:
Looks great. Makes much more sense now.